modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/idp/mgt/ResidentIDPConfigsTestCase.java

```diff
@@ -40,230 +44,209 @@
+                        "Expected Passive STS URL is not found in the tenant mode"},
+                {"wso2.com", "OpenIdUrl", "https://localhost:9853/openidserver",
+                        "Expected OpenID Server URL is not found in the tenant mode"},
+                {"wso2.com", "ECPUrl", "https://localhost:9853/samlecp?tenantDomain=wso2.com",
+                        "Expected ECP URL is not found in the tenant mode"},
+                {"wso2.com", "LogoutReqUrl", "https://localhost:9853/samlsso?tenantDomain=wso2.com",
+                        "Expected Logout URL is not found in the tenant mode"},
+                {"wso2.com", "ArtifactResolveUrl", "https://localhost:9853/samlartresolve",
+                        "Expected Artifact Resolution URL is not found in the tenant mode"},
+                {"wso2.com", "SSOUrl", "https://localhost:9853/samlsso?tenantDomain=wso2.com",
+                        "Expected SSO URL is not found in the tenant mode"},
+                {"wso2.com", "IDENTITY_PROVIDER_URL", "https://localhost:9853/services/t/wso2.com/wso2carbon-sts",
+                        "Expected Security Token Service URL is not found in the tenant mode"},
+                {"wso2.com", "OIDCWebFingerEPUrl", "https://localhost:9853/.well-known/webfinger",
+                        "Expected Web finger Endpoint URL is not found in the tenant mode"},
+                {"wso2.com", "OIDCCheckSessionEPUrl", "https://localhost:9853/oidc/checksession",
+                        "Expected Session IFrame Endpoint URL is not found in the tenant mode"},
+                {"wso2.com", "OAuth2IntrospectEPUrl", "https://localhost:9853/t/wso2.com/oauth2/introspect",
+                        "Expected Token Introspection Endpoint URL is not found in the tenant mode"},
+                {"wso2.com", "OAuth2RevokeEPUrl", "https://localhost:9853/oauth2/revoke",
+                        "Expected Token Revocation Endpoint URL is not found in the tenant mode"},
+                {"wso2.com", "OIDCLogoutEPUrl", "https://localhost:9853/oidc/logout",
+                        "Expected Logout Endpoint URL not found in the tenant mode"},
+                {"wso2.com", "OAuth2AuthzEPUrl", "https://localhost:9853/oauth2/authorize",
+                        "Expected Authorization Endpoint URL is not found in the tenant mode"},
+                {"wso2.com", "OAuth2DCREPUrl", "https://localhost:9853/t/wso2.com/api/identity/oauth2/dcr/v1.1/register",
+                        "Expected Dynamic Client Registration Endpoint URL is not found in the tenant mode"},
+                {"wso2.com", "OIDCDiscoveryEPUrl", "https://localhost:9853/t/wso2.com/oauth2/oidcdiscovery",
+                        "Expected Dynamic Client Registration Endpoint URL is not found in the tenant mode"},
+                {"wso2.com", "OAuth2JWKSPage", "https://localhost:9853/t/wso2.com/oauth2/jwks",
+                        "Expected is not found in the tenant mode"},
```

Comment:
Shall we mention what is not found here as well?

Reply:
Fixed 40f5178